### ISSUE TITLE
New Rule: function-no-unknown

### ISSUE THREAD

**Issue by SwetaTanwar** [eslint/css#147]
### Rule details

Disallow unknown functions.

### What type of rule is this?

Warns about a potential problem

### Example code

```css
The following patterns are considered problems:


a { transform: unknown(1); }


The following patterns are not considered problems:


a { transform: scale(1); }



a { transform: --custom-function(1); }
```

### Participation

- [x] I am willing to submit a pull request to implement this rule.

### Additional comments

Prior Art: https://stylelint.io/user-guide/rules/function-no-unknown/

**Comment by nzakas** [comment]
I assume this is inspired by the Stylelint rule? (Please be sure to mention prior art when suggesting a new rule.)

I'm not 100% sure, but I think `no-invalid-properties` might catch unknown functions. Can you double-check?

**Comment by nzakas** [comment]
Thanks for verifying @thecalamiity. Closing as this case is already covered.

As a note: Stylelint tends to have very narrowly focused rules whereas ESLint tends to have broader scope of rules. When we consider porting something from Stylelint it's helpful to double-check that we don't already have a rule that covers the same case.
